ci/Jenkinsfile

```diff
@@ -0,0 +1,148 @@
+pipeline {
+    agent any
+    options {
+        disableConcurrentBuilds(abortPrevious: true)
+    }
+    stages {
+        stage('Checks & Builds') {
+            failFast true
+            parallel {
+                stage('Check') {
+                    options {
+                        timeout(time: 1, unit: 'HOURS')
+                    }
+                    environment {
+                        HOME = "${WORKSPACE}"
+                    }
+                    agent {
+                        docker {
+                            image 'gpuci/rapidsai-driver:21.10-cuda11.4-devel-ubuntu20.04-py3.8'
+                            label 'cpu'
+                        }
+                    }
+                    steps {
+                        cleanWs(
+                            deleteDirs: true,
+                            externalDelete: 'sudo rm -rf %s'
```

Comment:
I don't know if `externalDelete` actually does anything. You could experiment with removing it if you'd like to make this less verbose.

Reply:
I tried removing this, the reason why this is needed appears to be a function of the docker pipeline, where some of the files are owned by root, while the `rm` is executed by the jenkins user.